static/templates/unstar_messages_modal.hbs

```diff
@@ -1,13 +0,0 @@
-<div id="unstar-messages-modal" class="modal new-style modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="unstar_messages_modal_label" aria-hidden="true">
-    <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-        <h3 id="unstar_messages_modal_label">{{t "Unstar all messages" }}</h3>
-    </div>
-    <div class="modal-body">
-        <p>{{t "Would you like to unstar all starred messages?  This action cannot be undone." }}</p>
-    </div>
-    <div class="modal-footer">
-        <button type="button" class="button rounded" data-dismiss="modal">{{t "Cancel" }}</button>
-        <button type="button" class="button rounded btn-danger" id="do_unstar_messages_button">{{t "Unstar all messages" }}</button>
-    </div>
-</div>
```

Comment:
It's nice to delete such a big block of code.  If you're looking for a mini-project, it's worth doing an audit for whether we can do this treatment in a separate PR to clean up other confirm dialogs.

Reply:
Yep! I'll try to do a follow up PR.